docker/Dockerfile.build.aarch64-linux

```diff
@@ -246,29 +312,30 @@ ARG DALI_BUILD_DIR=build_aarch64_linux
 
 WORKDIR /dali/${DALI_BUILD_DIR}
 
-CMD cmake \
-  -DWERROR=ON \
-  -DCMAKE_TOOLCHAIN_FILE:STRING="$PWD/../platforms/aarch64-linux/aarch64-linux.toolchain.cmake" \
-  -DCMAKE_COLOR_MAKEFILE=ON \
-  -DCMAKE_INSTALL_PREFIX=./install \
-  -DARCH=aarch64-linux \
-  -DCMAKE_CUDA_COMPILER=/usr/local/cuda-10.2/bin/nvcc \
-  -DCUDA_HOST=/usr/local/cuda-10.2 \
-  -DCUDA_TARGET=/usr/local/cuda-10.2/targets/aarch64-linux \
-  -DBUILD_TEST=ON \
-  -DBUILD_BENCHMARK=OFF \
-  -DBUILD_NVTX=OFF \
-  -DBUILD_PYTHON=OFF \
-  -DBUILD_LMDB=OFF \
-  -DBUILD_JPEG_TURBO=ON \
-  -DBUILD_LIBTIFF=ON \
-  -DBUILD_NVJPEG=OFF \
+CMD WERROR=ON           \
+    ARCH=aarch64-linux  \
+    BUILD_TEST=ON       \
+    BUILD_BENCHMARK=OFF \
+    BUILD_NVTX=OFF      \
+    BUILD_LMDB=ON       \
+    BUILD_JPEG_TURBO=ON \
+    BUILD_LIBTIFF=ON    \
+    BUILD_NVJPEG=OFF    \
   -DBUILD_NVJPEG2K=OFF \
-  -DBUILD_NVOF=OFF \
-  -DBUILD_NVDEC=OFF \
-  -DBUILD_LIBSND=ON \
-  -DBUILD_NVML=OFF \
-  -DBUILD_FFTS=ON \
-  -DVERBOSE_LOGS=OFF \
-  ..  && \
-  make -j"$(grep ^processor /proc/cpuinfo | wc -l)"
+    BUILD_NVOF=OFF      \
+    BUILD_NVDEC=OFF     \
+    BUILD_LIBSND=ON     \
+    BUILD_NVML=OFF      \
+    BUILD_FFTS=ON       \
+    VERBOSE_LOGS=OFF    \
+    TEST_BUNDLED_LIBS=NO\
+    WHL_PLATFORM_NAME=manylinux2014_aarch64 \
+    BUNDLE_PATH_PREFIX=aarch64-linux-gnu \
+    EXTRA_CMAKE_OPTIONS="-DCMAKE_TOOLCHAIN_FILE:STRING=$PWD/../platforms/aarch64-linux/aarch64-linux.toolchain.cmake \
+                        -DCMAKE_COLOR_MAKEFILE=ON \
+                        -DCMAKE_CUDA_COMPILER=/usr/local/cuda-10.2/bin/nvcc \
+                        -DCUDA_HOST=/usr/local/cuda-10.2 \
+                        -DCUDA_TARGET=/usr/local/cuda-10.2/targets/aarch64-linux" \
+    /dali/docker/build_helper.sh &&          \
+    rm -rf /dali/${DALI_BUILD_DIR}/nvidia* && \
+    cp -r /wheelhouse /dali/
```

Comment:
BTW, do we have a chance with producing a TF plugin as well? There is TF for Jeston, it would be nice if we could publish a wheel that works with it and advertise it.

Reply:
Plugin with no prebuild should work. But let us leave it to a separate PR.